src/common/header/Header.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 
+import ExternalLink, { Props as LinkProps } from '../externalLink/ExternalLink';
 import HelsinkiLogo from '../helsinkiLogo/HelsinkiLogo';
 import styles from './header.module.scss';
 
-const Header: React.SFC = () => {
+const Header = ({ href }: LinkProps) => {
   return (
     <header className={styles.header}>
-      <HelsinkiLogo color="white" />
+      <ExternalLink href={href}>
```

Comment:
This shouldn't be an external link.
Unlike `react-router-dom` links, normal (external) links force the page to refresh.

Reply:
do we really need routing here?

Reply:
I would say yes 🙂